src/Sherpa/ArticleViewController.swift

```diff
@@ -100,11 +112,21 @@ internal class ArticleViewController: ListViewController {
 			
 			let maxSize = CGSize(width: width - margins.left - margins.right, height: CGFloat.greatestFiniteMagnitude)
 			let titleSize = self.titleLabel.sizeThatFits(maxSize)
-			let bodySize = self.bodyView.sizeThatFits(maxSize)
-			
 			self.titleLabel.frame = CGRect(x: margins.left, y: 30, width: maxSize.width, height: titleSize.height)
-			self.bodyView.frame = CGRect(x: margins.left, y: self.titleLabel.frame.maxY + 15, width: maxSize.width, height: bodySize.height)
-			header?.frame = CGRect(x: 0, y: 0, width: width, height: self.bodyView.frame.maxY)
+            
+            if self.article.html.count > 0 {
+                let htmlSize = self.tableView.frame.height * 0.6    // Fixed height
```

Comment:
It is possible to get the content height of a `WKWebView` once the content has completed loading, so while this is an acceptable default, I would much rather have the actual content size be applied.